.sauce/config.yml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1alpha
 kind: puppeteer
 sauce:
   region: us-west-1
-  concurrency: 2 # Controls how many suites are executed at the same time.
+  concurrency: 100 # Controls how many suites are executed at the same time.
```

Comment:
Everything else looks good, but I was wondering why you'd want to set the default concurrency limit this high?

Reply:
@alexplischke to show our customers the power of concurrency. When doing demos it's one of the selling features for Sauce = massive parallelization. Look, I can run 100 tests in 15 seconds... Are you concerned about something?

Reply:
Well, the example only has two suites :D So only two will run in parallel anyway. I'm fine with raising the example, but it won't actually _do_ anything _to_ the example.